Makefile.include

```diff
@@ -5,6 +5,8 @@ EXTERNAL_BOARD_DIRS ?= $(MCUFIRMWAREBASE)/boards/
 LAST_MAKEFILEDIR = $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 RIOTBASE ?= $(MCUFIRMWAREBASE)/../RIOT/
 RIOTPKG ?= $(MCUFIRMWAREBASE)/pkg
+EXTERNAL_BOARD_DIRS ?= $(MCUFIRMWAREBASE)/boards/
+EXTERNAL_MODULE_DIRS += $(MCUFIRMWAREBASE)/drivers/lds01
```

Comment:
EXTERNAL_BOARD_DIRS is already set upper.

EXTERNAL_MODULE_DIRS will be set using MCUFW_MODULE_DIRS below.

Thus these 2 lines are useless.

Reply:
Done